src/backend/nodes/copyfuncs.c

```diff
@@ -1628,6 +1628,7 @@ _copyFuncExpr(const FuncExpr *from)
 	COPY_SCALAR_FIELD(inputcollid);
 	COPY_NODE_FIELD(args);
 	COPY_LOCATION_FIELD(location);
+	COPY_NODE_FIELD(context);
```

Comment:
Why can we have COPY_NODE_FIELD here but not in rel16?

Reply:
We have this in rel16, and it is automatically generated by `gen_node_support.pl`. What we don't have is read/write function call for this field.